cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/QueueAdmin.java

```diff
@@ -28,11 +28,6 @@
 public interface QueueAdmin {
 
   /**
-   * Deletes all entries for all queues.
-   */
-  void dropAll() throws Exception;
```

Comment:
Yes, nobody is using it (migrating to dropAllInNamespace).
in 2.6, we used to have dropAll(). now, we have dropAllInNamespace(namespaceId), for use from places like deleteNamespace(namespaceId)